ouroboros-consensus/src/Ouroboros/Consensus/Network/NodeToClient.hs

```diff
@@ -362,13 +407,15 @@ mkApps kernel Tracers {..} Codecs {..} Handlers {..} =
       bracketWithPrivateRegistry
         (chainSyncBlockServerFollower (getChainDB kernel))
         ChainDB.followerClose
-        $ \flr ->
+        $ \flr -> do
+          let cfg = topLevelConfigLedger $ getTopLevelConfig kernel
+          hflr <- newHistoryFollower cfg (getChainDB kernel)
```

Comment:
Note to self: this is just a closure-bound `TVar`, so no need for `registry`, `bracket` etc.